owncloudApp/src/main/java/com/owncloud/android/presentation/files/details/FileDetailsFragment.kt

```diff
@@ -309,23 +309,23 @@ class FileDetailsFragment : FileFragment() {
     }
 
     private fun setLastSync(ocFile: OCFile) {
-        if (ocFile.lastSyncDateForData!! > ZERO_MILLISECOND_TIME) {
+        if (ocFile.lastSyncDateForData?.let { it > ZERO_MILLISECOND_TIME } == true) {
```

Comment:
Unfortunately, It can´t be possible, in this line, you are trying to directly access `ocFile.lastSyncDateForData` and compare it with `ZERO_MILLISECOND_TIME`. If `lastSyncDateForData` is null, this will throw a `NullPointerException` error. In any case, Android studio already tells you that this comparison is not possible.